py3status/modules/scratchpad_counter.py

```diff
@@ -4,10 +4,11 @@
 
 Configuration parameters:
     cache_timeout: How often we refresh this module in seconds (default 5)
-    format: Format of indicator. {} replaces with count of windows
-        (default '{} ⌫')
+    format: string to print (default '{counter} ⌫')
```

Comment:
I don't like ` string to print` so if we sort that then I think this is good to merge